External/FEXCore/Source/Interface/Core/JIT/x86_64/MemoryOps.cpp

```diff
@@ -606,8 +636,14 @@ void JITCore::RegisterMemoryHandlers() {
   REGISTER_OP(STOREFLAG,           StoreFlag);
   REGISTER_OP(LOADMEM,             LoadMem);
   REGISTER_OP(STOREMEM,            StoreMem);
-  REGISTER_OP(LOADMEMTSO,          LoadMem);
-  REGISTER_OP(STOREMEMTSO,         StoreMem);
+  if (CTX->Config.TSOEnabled) {
```

Comment:
Why?
The IR op handling should be enabled regardless.
I could see this being useful for your debugging but it shouldn't hit upstream.

Reply:
Mostly to make it symmetrical with arm, and for validation. Never got hit though :P

Reply:
Done